src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -899,9 +901,12 @@ well as menu structures (for main menu and popup menus).
    <cmd id="reformatCode"
         menuLabel="Re_format Code"
         desc="Reformat the current line/selection"/>
-   <cmd id="lintActiveDocument"
+   <cmd id="showDiagnosticsActiveDocument"
         menuLabel="Show _Diagnostics"
         desc="Show diagnostics for the active document"/>
+   <cmd id="showDiagnosticsProject"
+        menuLabel="Show _Diagnostics (Project)"
```

Comment:
This needs a distinct mnemonic (_) character. The best way to figure out what's still available is to build the desktop version on Ubuntu then pull the Code menu down to see what characters are still left.